packages/dd-trace/src/plugins/util/cidr_matcher.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const semver = require('semver')
+
+if (semver.satisfies(process.version, '>=14.18.0')) {
+  const net = require('net')
+
+  module.exports = class CIDRMatcher {

```

Comment:
Can we do the opposite and return the built-in module directly when available, and then polyfill it using the library instead of the other way around?

Reply:
Done